github/store/teststore.go

```diff
@@ -0,0 +1,117 @@
+package store
+
+import (
+	"fmt"
+
+	"github.com/src-d/metadata-retrieval/github/graphql"
+)
+
+type TestStore struct {
+	organization *graphql.Organization
+	repository   *graphql.RepositoryFields
+	topics       []string
+	users        []*graphql.UserExtended
+	prs          []*graphql.PullRequest
+	prComments   []*graphql.IssueComment
+}
+
+func (s *TestStore) GetRepository() *graphql.RepositoryFields {
+	return s.repository
+}
+
+func (s *TestStore) GetOrganization() *graphql.Organization {
+	return s.organization
+}
+
+func (s *TestStore) GetNumOfPrs() int {
+	return len(s.prs)
+}
+
+func (s *TestStore) GetNumOfPrComments() int {
+	return len(s.prComments)
+}
+
+func (s *TestStore) GetNumOfUsers() int {
+	return len(s.users)
+}
+
+func (s *TestStore) SaveOrganization(organization *graphql.Organization) error {
+	fmt.Printf("organization data fetched for %s\n", organization.Login)
```

Comment:
according to logging please consider using the logger from our code convention [here](https://github.com/src-d/guide/blob/806661020bf47a2a736e7b72aee1095d42a41774/engineering/conventions/go.md#logging)